tasks/main.yml

```diff
@@ -5,11 +5,11 @@
 - include: setup-Debian.yml
   when: ansible_os_family == 'Debian'
 
-- name: Install Elasticsearch Curator and required dependencies.
-  pip: "name={{ item }}"
-  with_items:
-    - elasticsearch-curator
-    - argparse
+- name: Install Elasticsearch Curator dependencies.
+  pip: name=argparse
+
+- name: Install Elasticsearch Curator.
+  pip: name=elasticsearch-curator version=3.5.1
```

Comment:
We can still keep these in the same task, but regardless of how we split up the tasks, I'd like to make the version configurable. Maybe we could break out elasticsearch curator's version into a variable or the set of `with_items` as a dict, like I do for my Composer role here: https://github.com/geerlingguy/ansible-role-composer/blob/master/defaults/main.yml#L13